storage/scratchpad/benches/sparse_merkle.rs

```diff
@@ -167,9 +164,9 @@ impl Benches {
         rng: &mut StdRng,
         keys: &[HashValue],
         block_size: usize,
-    ) -> Vec<Vec<(HashValue, Option<StateValue>)>> {
-        std::iter::repeat_with(|| vec![Self::gen_update(rng, keys), Self::gen_update(rng, keys)])
-            .take(block_size)
+    ) -> Vec<(HashValue, Option<StateValue>)> {
+        std::iter::repeat_with(|| Self::gen_update(rng, keys))
+            .take(block_size * 2)
```

Comment:
I think it's more straightforward not to x2

Reply:
Fixed.